src/Stubs/Doctrine/StubFilesExtensionLoader.php

```diff
@@ -59,6 +61,13 @@ public function getFiles(): array
 			$files[] = $stubsDir . '/ServiceEntityRepository.stub';
 		}
 
+		$collectionVersion = InstalledVersions::getVersion('doctrine/dbal');
```

Comment:
Strictly speaking, we should even catch OutOfBoundsException in case dbal is not installed: https://getcomposer.org/doc/07-runtime.md#knowing-the-version-of-package-x

Reply:
Oh damn, and the check should be about `'doctrine/collections'` not `'doctrine/dbal'`.

I fixed it and added a class_exists check and the try/catch of OutOfBoundsException.